src/api/roles/shared/chat.py

```
@@ -0,0 +1,83 @@
+def create_new_chat(
+    payload: CreateNewChatInput,
+    db = Depends(get_session),
+    acc: Account = Depends(get_account_from_bearer)
+):
+    #checking if relationship is valid
+    relationship = db.get(ClientCoachRelationship, payload.relationship_id)
+    
+    if relationship is None:
+        raise HTTPException(404, detail="Relationship not found")
+    
+    if relationship.is_active == False:
```

Ruff (0.15.10)

[error] Avoid equality comparisons to `False`; use `not relationship.is_active:` for false checks

Replace with `not relationship.is_active`

(E712)